distrho/src/DistrhoPluginVST.cpp

```diff
@@ -463,7 +463,7 @@ class PluginVst
                 }
                 else if (hints & kParameterIsInteger)
                 {
-                    DISTRHO_NAMESPACE::snprintf_param((char*)ptr, (int32_t)value, 24);
+                    DISTRHO_NAMESPACE::snprintf_iparam((char*)ptr, (int32_t)(value + 0.5f), 24);
```

Comment:
Actually, this won't work with negative parameters. Shouldn't this be (int32_t)std::round(value) ?